packages/components/src/molecules/ProductCard/ProductCardContent.tsx

```diff
@@ -16,6 +16,10 @@ import {
   Rating,
 } from '../../'
 
+type DeliveryBadge = {
```

Comment:
suggestion: thinking about those `DeliveryBadge` and `deliveryBadges`... we could use `DeliveryPromiseBadge`/`deliveryPromiseBadges` instead to be clearer (or `availabilityBadges` for example, but I believe that linking it to DP would be better)